src/cocotb_tools/runner.py

```diff
@@ -159,10 +173,33 @@ def build(
     ) -> None:
         """Build the HDL sources.
 
+        With mixed language simulators, *sources* will be built,
+        followed by *vhdl_sources*, then *verilog_sources*.
```

Comment:
In the ASIC world, low-level models (cell primitives from a PDK) are usually Verilog, while higher levels are VHDL or Verilog, so changing the order (or being able to change it via a switch) would be helpful in that context.

Reply:
vhdl_sources and verilog_sources will be deprecated by a follow-on PR. The intent is to only use `sources` going forward.